roles/velero/templates/registry_aws.yml.j2

```diff
@@ -0,0 +1,84 @@
+---
+{% for bsl in backup_storage_locations %}
+{% if bsl.provider == 'aws' %}
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  namespace: {{ velero_namespace }}
+  name: aws-registry
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      component: aws-registry
+  template:
+    metadata:
+      labels:
+        component: aws-registry
+    spec:
+      restartPolicy: Always
+      containers:
+      - image: quay.io/konveyor/registry:latest
+        name: aws-registry-container
+        ports:
+        - containerPort: 5000
+          protocol: TCP
+        env:
+          - name: REGISTRY_STORAGE
+            value: s3
+          - name: REGISTRY_STORAGE_S3_ACCESSKEY
+            value: "{{ aws_access_key_id }}"
+          - name: REGISTRY_STORAGE_S3_BUCKET
+            value: "{{ bsl.object_storage.bucket }}"
+          - name: REGISTRY_STORAGE_S3_REGION
+            value: "{{ bsl.config.region }}"
+          - name: REGISTRY_STORAGE_S3_SECRETKEY
+            value: "{{ aws_secret_access_key }}"
+{% if bsl.config.s_3__url is defined %}
+          - name: REGISTRY_STORAGE_S3_REGIONENDPOINT
+            value: {{ bsl.config.s_3__url }}
+{% endif %}
+{% if bsl.config.root_directory is defined %}
+          - name: REGISTRY_STORAGE_S3_ROOTDIRECTORY
+            value: {{ bsl.config.root_directory }}
+{% endif %}
+{% if bsl.config.insecure_skip_tls_verify is defined %}
+          - name: REGISTRY_STORAGE_S3_SKIPVERIFY
+            value: "{{ bsl.config.insecure_skip_tls_verify | string }}"
+{% endif %}
+
+---
+apiVersion: v1
+kind: Service
+metadata:
+  labels:
+    component: aws-registry
```

Comment:
NIT: probably good to rename this `oadp-aws-registry`

Reply:
Updated it to `oadp-<bls.name>-<bsl.provider>-registry`